server/log_rpc_server_test.go

```diff
@@ -292,8 +292,9 @@ func TestQueueLeaves(t *testing.T) {
 	if queuedLeaf.Status != nil && queuedLeaf.Status.Code != int32(code.Code_OK) {
 		t.Errorf("QueueLeaves().Status=%d,nil; want %d,nil", queuedLeaf.Status.Code, code.Code_OK)
 	}
-	if !reflect.DeepEqual(queueRequest0.Leaves[0], queuedLeaf.Leaf) {
-		t.Errorf("QueueLeaves()=%+v,nil; want %+v,nil", queuedLeaf, queueRequest0.Leaves)
+	if !proto.Equal(queueRequest0.Leaves[0], queuedLeaf.Leaf) {
```

Comment:
Not sure of the reason for this structure. Is pretty.Compare() expected to behave differently to proto.Equal() or is it for performance reasons maybe?